src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -163,18 +164,43 @@ export function CreateExportModal({
             {" "}
             {issuesToExport.length}
           </Text>
-          <Button
-            ml="auto"
-            size="sm"
-            onClick={() => {
-              exportIssues(
-                issuesToExport,
-                issueStatus.filter((s) => includedIssueStatus.includes(s.name)),
-              );
-            }}
+          <Tooltip
+            withArrow
+            multiline
+            w={200}
+            fz={14}
+            fw={500}
+            openDelay={200}
+            closeDelay={200}
+            opened={
+              (endDate === null
+                || startDate === null
+                || issuesToExport.length === 0)
+              && hoveredoverexport
+            }
+            ta="center"
+            color="primaryBlue"
+            variant="filled"
+            label="Please select Issue Types and Status with a date range with at least 1 issue to export"
           >
-            Export CSV
-          </Button>
+            <Button
+              ml="auto"
+              size="sm"
+              disabled={endDate === null
+                || startDate === null
+                || issuesToExport.length === 0}
```

Comment:
If you base this PR on https://github.com/MaibornWolff/ProjectCanvas/pull/128, you can simplify this to `issuesToExport.length === 0` since issuesToExport wont be filled unless both dates are set.